client/extensions/woocommerce/index.js

```diff
@@ -64,6 +64,12 @@ const getStorePages = () => {
 			path: '/store/orders/:site',
 		},
 		{
+			container: Orders,
+			configKey: 'woocommerce/extension-orders',
+			documentTitle: translate( 'Orders' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **100** times:
`translate( 'Order' )` *ES Score: 6*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/orders-status-filtering)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).